scania-digital-design-system/tegel
# test(link): added unit test

**Describe pull-request**  
Adding unit tests for Link component

**Solving issue**  
Fixes: [CDEP-3012](https://tegel.atlassian.net/browse/CDEP-3012)

**How to test**  
1. Check [unit test spreadsheet for Link component](https://scaniaazureservices-my.sharepoint.com/:x:/r/personal/tatyana_bumazhkina_scania_com/Documents/Unit%20test%20scenarios.xlsx?d=wca48bf6d92cf4e8c8f456337e2002ec4&csf=1&web=1&e=zOXTNJ&nav=MTVfe0VCNDI3Rjc2LUEzMTQtNEM4Ri1CQkMyLUMxMzE2MTY3NTJDN30).
2. Make sure unit tests in PR do the checks written in the form above 
3. Run code on your local to see that tests pass
